src/global_controllers.h

```diff
@@ -0,0 +1,37 @@
+#ifndef GLOBALS_HPP
```

Comment:
Instead of using your own include guard, you can use:
`#pragma once`
This is already used in this project.